[
    {
        "content": "<p>Hello! When I run `mvn -Pct clean package docker:run I am receiving the following error:      </p>\n<p>[<strong>ERROR</strong>] DOCKER&gt; Unable to pull 'solr:9.4.1' : {\"message\":\"failed to resolve reference \\\"<a href=\"http://docker.io/library/solr:9.4.1\\\">docker.io/library/solr:9.4.1\\</a>\": failed to authorize: failed to fetch oauth token: unexpected status from GET request to <a href=\"https://auth.docker.io/token?scope=repository%3Alibrary%2Fsolr%3Apull&amp;service=registry.docker.io\">https://auth.docker.io/token?scope=repository%3Alibrary%2Fsolr%3Apull&amp;service=registry.docker.io</a>: 401 Unauthorized\"} (Internal Server Error: 500) [{\"message\":\"failed to resolve reference \\\"<a href=\"http://docker.io/library/solr:9.4.1\\\">docker.io/library/solr:9.4.1\\</a>\": failed to authorize: failed to fetch oauth token: unexpected status from GET request to <a href=\"https://auth.docker.io/token?scope=repository%3Alibrary%2Fsolr%3Apull&amp;service=registry.docker.io\">https://auth.docker.io/token?scope=repository%3Alibrary%2Fsolr%3Apull&amp;service=registry.docker.io</a>: 401 Unauthorized\"} (Internal Server Error: 500)]</p>\n<p>Has anyone come across this issue and know how to get past this?</p>",
        "id": 474045008,
        "sender_full_name": "Omer M Fahim",
        "timestamp": 1727791026
    },
    {
        "content": "<p>Does this also happen to you when doing the pull manually? <code>docker pull docker.io/library/solr:9.4.1</code></p>",
        "id": 474047761,
        "sender_full_name": "Oliver Bertuch",
        "timestamp": 1727791770
    },
    {
        "content": "<p>If that doesn't work, I recommend to use <code>docker login</code> before starting to pull materials and try again</p>",
        "id": 474047849,
        "sender_full_name": "Oliver Bertuch",
        "timestamp": 1727791794
    },
    {
        "content": "<p>when executing the pull manually I receive a similar error:     </p>\n<p>View a summary of image vulnerabilities and recommendations → docker scout quickview <a href=\"http://docker.io/library/solr:9.4.1\">docker.io/library/solr:9.4.1</a></p>\n<p>Error response from daemon: failed to resolve reference \"<a href=\"http://docker.io/library/solr:9.4.1\">docker.io/library/solr:9.4.1</a>\": failed to authorize: failed to fetch oauth token: unexpected status from GET request to <a href=\"https://auth.docker.io/token?scope=repository%3Alibrary%2Fsolr%3Apull&amp;service=registry.docker.io\">https://auth.docker.io/token?scope=repository%3Alibrary%2Fsolr%3Apull&amp;service=registry.docker.io</a>: 401 Unauthorized</p>\n<p>Also, I was logged in to the account.</p>",
        "id": 474066792,
        "sender_full_name": "Omer M Fahim",
        "timestamp": 1727796872
    },
    {
        "content": "<p>I was able to resolve this issue. Essentially what I did was uninstall Docker Desktop, install latest version, and signing out of my docker account.</p>",
        "id": 474345514,
        "sender_full_name": "Omer M Fahim",
        "timestamp": 1727886806
    },
    {
        "content": "<p><span class=\"user-mention silent\" data-user-id=\"598183\">Oliver Bertuch</span> has marked this topic as resolved.</p>",
        "id": 474525003,
        "sender_full_name": "Notification Bot",
        "timestamp": 1727945273
    }
]